3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -686,8 +686,98 @@ L'ambiente tridimensionale è realizzato mediante i componenti:
 
 - *Warehouse*: elemento che contiene la logica principale della visualizzazione dell'ambiente grafico ed il canvas con gli elementi 3D.
 
+\
+= Archietettura di deployment
+Nel contesto del progetto didattico, l'adozione di un'architettura monolitica è stata determinata valutando fattori strategici in termini di risorse, tempi ed esperienza del gruppo.
+
+== Analisi dell'architettura monolitica
+L'architettura monolitica rappresenta un modello di progettazione architetturale in cui tutte le funzionalità e i componenti del prodotto software risiedono in un unico sistema autocontenuto e indipendente.
+
+Nel contesto del progetto didattico, l'architettura monolitica è stata scelta tenendo conto dei seguenti vantaggi e considerazioni:
+
+- *Gestione di una singola unità*: l'intero set di funzionalità risisede in unico sistema. Questo approccio semplifica lo sviluppo e la manutenzione del codice, rimuovendo la complessità associata alla comunicazione tra servizi e alla gestione dei dati distribuiti in un'architettura distribuita;
```

Comment:
- "... l'intero set di funzionalità **risiede** ...";
- "distribuiti in un'architettura distribuita" è un po' ripetitivo. Propongo "gestione dei dati distribuiti tipica di un'architettura non monolitica.", ma se trovi un modo migliore top.